src/ApiFunctions/GetIssuesByProject.ts

```diff
@@ -0,0 +1,93 @@
+export interface Issue {
+    project: string,
+    id: string,
+    summary: string,
+    description?: string,
+    assignee?: {
+      displayName: string,
+      accountId: string
+    },
+    reporter?: {
+      displayName: string,
+      accountId: string
+    },
+    priority: "Highest" | "High" | "Medium" | "Low" | "Lowest",
+    created: string,
+    duedate?: string,
+    timespent?: number
+};
+
+//Pulls Issue data of a given project and returns a promise with the data in a useable format
+export default async function getIssuesByProject(projectKey: string) : Promise<Issue[]> {
+    require('dotenv').config();
+    const username: string | undefined = process.env.ATLASSIAN_USERNAME;
+    const password: string | undefined  = process.env.ATLASSIAN_API_KEY;
+    const domain: string | undefined  = process.env.DOMAIN;
+    let doneBool = false;
+    let result: Issue[] = [];
+    let startAt = 0;
+
+    while (!doneBool) {
+        const bodyData = `{
+            "fields": [
+                "summary",
+                "description",
+                "assignee",
+                "reporter",
+                "priority",
+                "duedate",
+                "created",
+                "timespent"
+            ],
+            "jql": "project = ${projectKey}",
+            "startAt": "${startAt}"
+        }`;
+        try {
+            let response = await fetch(`https://${domain}.atlassian.net/rest/api/3/search`, {
```

Comment:
why let? can not we use const here?

If something is not changing or immutable. please have that as const instead of let.